astropy/units/core.py

```diff
@@ -261,6 +272,48 @@ def add_enabled_equivalencies(self, equivalencies):
         equivalencies = _normalize_equivalencies(equivalencies)
         self._equivalencies |= set(equivalencies)
 
+    def set_enabled_aliases(self, aliases):
+        """
+        Set aliases for units, e.g., to handle alternate spellings.
+
+        Parameters
+        ----------
+        aliases : dict of str, Unit
+            The aliases to set. The keys must be the string aliases, and values
+            must be the `astropy.units.Unit` that the alias will be mapped to.
```

Comment:
```suggestion
            must be the `astropy.units.Unit` that the alias will be mapped to. For example, ``{"parsnek": u.parsec}``.
```
(actual example might be too much of a joke but I think it would be good to have some small example here?

Reply:
I'd tend to just leave it to the more commonly used functions, which I did give an example.